csv/stats.csv

```diff
@@ -0,0 +1,5 @@
+Name,Total,Pass,Time
+is_sized,20200,10185,0.3s
+is_sorted,20200,2957,0.3s
+is_duplicate,20200,2188,0.3s
+is_unique,20200,8074,0.7s
```

Comment:
So is the file name the generator used and then the contents is checking that generator against each precondition? Interesting to think about the relationships here.

If so, what is stats.csv?

Reply:
I think the conclusion was that this file could be better named.